lib/profiler.js

```diff
@@ -82,7 +83,8 @@ function getUsedNodeModules() {
   );
 }
 
-function writeProfileFiles(profilePath, options) {
+function writeProfileFiles(options) {
+  const { profilePath, open, ...opts } = options;
```

Comment:
This will break node 6 support (hopefully everyone has upgraded already but I don't we should destroy the support just for spread)

Reply:
I didn't notice Node v6 is still supported -- it has been EOL for a long while.

will update later!

Reply:
cool thanks :)